src/components/registerV2/registerV2.css

```diff
@@ -1,7 +1,9 @@
 @import './../app/variablesV2.css';
+@import './../app/mixins.css';
 
 :root {
   --registration-title-font-size: 24px;
+  --title-color: #0c152e;
```

Comment:
update to correct color and since you are using the mixin, you can remove the font-size.